### DISCUSSION TITLE
Monofur Nerd Font not working

### DISCUSSION THREAD

**Discussion by BraxtonMDevelops** [wezterm/wezterm#719] [Q&A] [answered]
Monofur Nerd Font (the main font i use) is not working on wezterm . it works on other applications such as emacs, or kitty. It is worth noting here that other nerdfonts such as meslolgl work but not this one. Thanks for your help

**Comment by wez** [comment, marked as answer]
Would you mind opening a bug report for this, rather than a discussion? The new bug flow asks you for information that will help me understand what version and environment you're running in, and is easier for me to manage the state of your issue.  Thanks!

**Reply by wez** [reply]
Also: this may be a duplicate of https://github.com/wez/wezterm/issues/726
